<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>280</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>360</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tournie</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-TapBoard-Desktop_Qt_5_7_0_MSVC2015_32bit-Debug/images/sourceimgs/0icon.png</normaloff>../build-TapBoard-Desktop_Qt_5_7_0_MSVC2015_32bit-Debug/images/sourceimgs/0icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>339</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="main_L">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="title_L">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="titleBox">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="setType_L">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="setTypeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Set type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="setTypeSDBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Singles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Doubles</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="setTypeCheckBox">
         <property name="toolTip">
          <string>Displays Singles/Doubles. Uncheck to not show.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="setTypeWLBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Winners Bracket</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Losers Bracket</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Winners Quarter-Finals</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Losers Quater-Finals</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Winners Semi-Finals</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Losers Semi-Finals</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Winners Finals</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Losers Finals</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Grand Finals</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="playerName_L">
       <item>
        <widget class="QLineEdit" name="nameOneBox">
         <property name="text">
          <string>Player 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="vsLabel">
         <property name="text">
          <string>VS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameTwoBox">
         <property name="text">
          <string>Player 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="playerCharChange_L">
       <item>
        <layout class="QVBoxLayout" name="playerOneChar_L">
         <item>
          <widget class="QLabel" name="playerOneLabel">
           <property name="text">
            <string>Player 1 Character:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="charOneBox">
           <item>
            <property name="text">
             <string>Fox</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sheik</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Falco</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Marth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jigglypuff</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Peach</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Captain Falcon</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ice Climbers</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Samus</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pikachu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dr. Mario</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ganondorf</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Luigi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mario</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Young Link</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yoshi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Link</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Donkey Kong</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Roy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mr. Game and Watch</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mewtwo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ness</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Zelda</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pichu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bowser</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Kirby</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="sliderOne_L">
           <item>
            <widget class="QSlider" name="skinSliderOne">
             <property name="maximum">
              <number>3</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sliderImageOne">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../build-TapBoard-Desktop_Qt_5_7_0_MSVC2015_32bit-Debug/images/characters/fox/fox0.png</pixmap>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="scoreOne_L">
           <item>
            <widget class="QLabel" name="playerOneSLabel">
             <property name="text">
              <string>Player 1 Score:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="scoreOneBox">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="playerTwoChar_L">
         <item>
          <widget class="QLabel" name="playerTwoLabel">
           <property name="text">
            <string>Player 2 Character:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="charTwoBox">
           <item>
            <property name="text">
             <string>Fox</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sheik</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Falco</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Marth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jigglypuff</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Peach</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Captain Falcon</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ice Climbers</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Samus</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pikachu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dr. Mario</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ganondorf</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Luigi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mario</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Young Link</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yoshi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Link</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Donkey Kong</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Roy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mr. Game and Watch</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mewtwo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ness</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Zelda</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pichu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bowser</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Kirby</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="sliderTwo_L">
           <item>
            <widget class="QSlider" name="skinSliderTwo">
             <property name="maximum">
              <number>3</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sliderImageTwo">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../build-TapBoard-Desktop_Qt_5_7_0_MSVC2015_32bit-Debug/images/characters/fox/fox0.png</pixmap>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="scoreTwo_L">
           <item>
            <widget class="QLabel" name="playerTwoSLabel">
             <property name="text">
              <string>Player 2 Score:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="scoreTwoBox">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="button_L">
       <item>
        <widget class="QPushButton" name="swapButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Swap</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="actionSelect_game_coming_soon"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save-As..</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset to defaults</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionSelect_game_coming_soon">
   <property name="text">
    <string>Select game...(coming soon)</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>charOneBox</tabstop>
  <tabstop>charTwoBox</tabstop>
  <tabstop>skinSliderTwo</tabstop>
  <tabstop>scoreTwoBox</tabstop>
  <tabstop>swapButton</tabstop>
  <tabstop>updateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
